protoc-gen-openapiv2/internal/genopenapi/template.go

```diff
@@ -581,6 +582,10 @@ func schemaOfField(f *descriptor.Field, reg *descriptor.Registry, refs refMap) o
 		updateSwaggerObjectFromFieldBehavior(&ret, j, f)
 	}
 
+	if reg.GetOptionalNullable() && f.Proto3Optional != nil && *f.Proto3Optional {
```

Comment:
Can we just use the getter?
```suggestion
	if reg.GetOptionalNullable() && f.GetProto3Optional() {
```

Reply:
Yep, I was using completion and missed the presence of the getters. Updated.

Reply:
Yep, I was using completion and missed the presence of the getters. Updated.